quick-start.md

```diff
@@ -13,15 +13,61 @@ has_toc: false
 
 ### Overview
 
-Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate model sync service.  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
+Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate [model sync service](https://open-horizon.github.io/docs/developing/model_management_details/#model_management_details).  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
 
-To install a simple, developer-friendly version of [all the services on one device](https://open-horizon.github.io/docs/mgmt-hub/docs/), run the following one-liner as `root` on an x86_64 machine running Ubuntu 18.04:
+Open Horizon allows you to install individual tools like the Management Hub control plane, the CLI for remote management using the command-line, or the anax Agent.  If you would like all three installed on a single machine, use the all-in-one installation provided below.
+
+### All-in-one installation instructions
+
+To install a simple, developer-friendly version of [all the services on one device](https://open-horizon.github.io/docs/mgmt-hub/docs/), run the following one-liner as `root` on an x86_64 machine running Ubuntu 18.04 (see [all supported environments](https://open-horizon.github.io/docs/installing/adding_devices/#suparch-horizon)):
```

Comment:
Don't use the full URL.  Use relative paths in both of these links.

Reply:
fixed